ecosystem/indexer-grpc/indexer-grpc-utils/src/config.rs

```diff
@@ -8,7 +8,7 @@ use std::path::PathBuf;
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct GcsFileStore {
     pub gcs_file_store_bucket_name: String,
-    // TODO: consider adding credentials_path, rather than relying on default location
+    pub gcs_file_store_service_account_key_path: String,
```

Comment:
maybe make this `Option`, so we can default to the default location if unspecified?

Reply:
the problem is, if SERVER_ACCOUNT is not set, there is no default value and the component will panic.. Let's force users to set the service account if they're using gcs.

Reply:
I see ok!